SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -208,6 +210,15 @@ class FReceptionistConfig : public FConnectionConfig
 		}
 	}
 
+	void SetReceptionistHost(const FURL& URL)
```

Comment:
Perhaps we can add in here a check: 

if (!URL.Host.IsEmpty())
{
    SetReceptionistHost(URL.Host);
}

Then we can call FReceptionistConfig::SetReceptionistHost(const FURL& URL) from USpatialConnectionManager::SetupConnectionConfigFromURL.

Reply:
Huh, totally true, didn't see that. 👍